---
content_type: page
title: Readings
uid: ac73e43e-a545-069f-5154-59eef6759d00
---

Part I: Defense Mechanisms Developed by Unicellular Organisms
-------------------------------------------------------------

To protect themselves, organisms, starting from prokaryotes, have developed mechanisms for the recognition and elimination of pathogens. Recognition mechanisms can be based on either the presence of distinctive foreign markers or by the absence of "self" markers. Accordingly, mechanisms of pathogen recognition as well as self/non-self discrimination have evolved.

Part II: The Transition to Multi-Cellularity Resulted in Cell Specialization
----------------------------------------------------------------------------

Many protective mechanisms of unicellular organisms were preserved in all cells of multi-cellular organisms, whereas others were preserved only in specialized cells. The transition to multi-cellularity necessitated not only recognition of pathogens but also recognition of an organism's own cells altered by pathological processes, such as cancer. Immune cells specialized in recognition and elimination of foreign invaders and compromised self-cells have evolved.

Part III: An Evolutionary Milestone: The Origin of an Adaptive Immune System
----------------------------------------------------------------------------

All previously discussed mechanisms (often called "innate immunity") are based on pathogen-recognizing receptors encoded in the genome. About 450-470 million years ago, vertebrates developed a new system of pathogen recognition. This system is based on the creation of unlimited variability of immune receptors and on clonal expansion of cells bearing a specific receptor in response to an antigen/pathogen challenge. This mechanism allows for recognition and protection against unlimited variety of antigens/pathogens.

| WEEKÂ # | TOPICS | OVERVIEW | READINGS |
| --- | --- | --- | --- |
|  {{< br >}}{{< br >}} 1 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Introduction of Instructor and Students {{< br >}}{{< br >}} Overview and Discussion of the Syllabus {{< br >}}{{< br >}} Aim of the Course {{< br >}}{{< br >}} How to Read a Scientific Paper {{< br >}}{{< br >}} Literature and Database Searching {{< br >}}{{< br >}} Introduction to Next Week's Topic {{< br >}}{{< br >}}  | &nbsp; |
| {{< td-colspan 4 >}} {{< br >}}{{< br >}} **Part I: Defense Mechanisms Developed by Unicellular Organisms** {{< br >}}{{< br >}} {{< /td-colspan >}} ||||
|  {{< br >}}{{< br >}} 2 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Recognition of Foreign Nucleic Acids {{< br >}}{{< br >}} The Restriction/Modification (R-M) System {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} The restriction/modification (R-M) system of bacteria recognizes the absence of self as a feature of a pathogen. Host DNA is protected from digestion by methylation at specific sequences, while non-methylated foreign DNA is destroyed by restriction enzymes. This system possibly originated from the bacterial mismatch repair system, which is also based on recognition of the degree of DNA methylation. The principle of recognition of DNA methylation as a means of self/non-self discrimination was successful throughout evolution; thus immune cells of vertebrates recognize methylation patterns of bacterial DNA as non-self. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Meselson, M., and R. Yuan. "DNA restriction enzyme from E. coli." _Nature_ 217 (1968): 1110-4. {{< br >}}{{< br >}} Hemmi, H., O. Takeuchi, T. Kawai, T. Kaisho, S. Sato, H. Sanjo, M. Matsumoto, K. Hoshino, H. Wagner, K. Takeda, and S. Akira. "A Toll-like receptor recognizes bacterial DNA." _Nature_ 408 (2000): 740-5. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 3 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Recognition of Foreign Nucleic Acids {{< br >}}{{< br >}} RNAi {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Another protective mechanism against pathogenic nucleic acids is RNA interference. It is conserved from unicellular eukaryotes to mammals and is based on recognition of double-stranded RNA. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Hamilton, A. J., and D. C. Baulcombe. "A species of small antisense RNA in posttranscriptional gene silencing in plants." _Science_ 286 (1999): 950-2. {{< br >}}{{< br >}} Li, H., W. X. Li, and S. W. Ding. "Induction and suppression of RNA silencing by an animal virus." _Science_ 296 (2002): 1319-21. {{< br >}}{{< br >}} _Recommended Reading_ {{< br >}}{{< br >}} Plasterk, R. H. "RNA silencing: the genome's immune system." _Science_ 296 (2002): 1263-5. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 4 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Anti-Microbial Peptides {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Anti-microbial peptides are found throughout all phyla and play a fundamental role in anti-microbial protection. Anti-microbial peptides target a distinctive feature of microbial membranes, namely, negatively charged molecules on the membranes' outer sides. Despite the great variety of these peptides (about 900 known), their fundamental structural principle known as an 'amphipathic' design (clusters of hydrophobic and cationic amino acids organized in discrete sectors) has been maintained in evolution. Such evolutionary distant anti-microbial peptides as amoebapores of amoeba and mammalian granulysins are thought to be homologous. The same design might also have originated by convergent evolution. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Lemaitre, B., J. M. Reichhart, and J. A. Hoffmann. "Drosophila host defense: differential induction of antimicrobial peptide genes after infection by various classes of microorganisms." _Proc Natl Acad Sci USA_ 94 (1997): 14614-9. {{< br >}}{{< br >}} Stenger, S., D. A. Hanson, R. Teitelbaum, P. Dewan, K. R. Niazi, C. J. Froelich, T. Ganz, S. Thoma-Uszynski, A. Melian, C. Bogdan, S. A. Porcelli, B. R. Bloom, A. M. Krensky, and R. L. Modlin. "An antimicrobial activity of cytolytic T cells mediated by granulysin." _Science_ 282 (1998):121-5. {{< br >}}{{< br >}} _Recommended Reading_ {{< br >}}{{< br >}} Zasloff, M. "Antimicrobial peptides of multicellular organisms." _Nature_ 415 (2002): 389-95. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 5 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Altruistic Death {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Altruistic death can be regarded as an example of convergent evolution, where bacteria and eukaryotic cells independently developed a mechanism of trapping a pathogen inside the infected cell. As a result, the cell dies, killing the pathogen inside and thereby benefiting the rest of the cell population.  {{< br >}}In bacteria, this mechanism is known as abortive infection system (Abi). In multi-cellular organisms, an infected cell can undergo programmed cell death or apoptosis. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Bouchard, J. D., E. Dion, F. Bissonnette, and S. Moineau. "Characterization of the two-component abortive phage infection mechanism AbiT from Lactococcus lactis." _J Bacteriol_ 184 (2002): 6325-32. {{< br >}}{{< br >}} Tollefson, A. E., T. W. Hermiston, D. L. Lichtenstein, C. F. Colle, R. A. Tripp, T. Dimitrov, K. Toth, C. E. Wells, P. C. Doherty, and W. S. Wold. "Forced degradation of Fas inhibits apoptosis in adenovirus-infected cells." _Nature_ 392 (1998): 726-30. {{< br >}}{{< br >}} _Recommended Reading_ {{< br >}}{{< br >}} Raff, M. "Cell suicide for beginners." _Nature_ 396 (1998): 119-22. {{< br >}}{{< br >}}  |
| {{< td-colspan 4 >}} {{< br >}}{{< br >}} **Part II: The Transition to Multi-Cellularity Resulted in Cell Specialization** {{< br >}}{{< br >}} {{< /td-colspan >}} ||||
|  {{< br >}}{{< br >}} 6 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} How are pathogens recognized by multi-cellular organisms? {{< br >}}{{< br >}} Pattern Recognition and Missing Self {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} The two principles of detecting pathogens, by recognition of pathogen markers or by recognition of "missing self", have evolved to greater complexity in multi-cellular organisms.  {{< br >}}Many receptors have evolved for recognizing conserved molecular patterns characteristic of microbial pathogens, e.g. macrophage mannose-binding receptor recognizes gram- positive and negative bacteria and fungi. Mannose-binding was preserved in evolution, since phagotrophic amoebae also use mannose receptors to recognize bacteria.  {{< br >}}To recognize a pathogen as "missing-self", the organism marks its own cells by expressing specific molecule/s and then screening all cells for the presence of this tag. Any cell that has failed the screen is destroyed. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Ezekowitz, R. A., M. Kuhlman, J. E. Groopman, and R. A. Byrn. "A human serum mannose-binding protein inhibits in vitro infection by the human immunodeficiency virus." _J Exp Med_ 169 (1989): 185-96. {{< br >}}{{< br >}} Stern, P., M. Gidlund, A. Orn, and H. Wigzell. "Natural killer cells mediate lysis of embryonal carcinoma cells lacking MHC." _Nature_ 285 (1980): 341-2. {{< br >}}{{< br >}} _Recommended Reading_ {{< br >}}{{< br >}} Medzhitov, R., and C. A. Janeway, Jr. "Decoding the patterns of self and non-self by the innate immune system." _Science_ 296 (2002): 298-300. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 7 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} How do multi-cellular organisms destroy pathogens? {{< br >}}{{< br >}} Phagocytosis {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Unicellular organisms developed phagocytosis as a trophic mechanism. Multi-cellular organisms transformed this mechanism into protection against endogenous and exogenous pathogens. Specialized phagocytes appear in sponges, the evolutionary oldest metazoan. In higher organisms several phagocytosing cell types evolved. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Tsukano, H., F. Kura, S. Inoue, S. Sato, H. Izumiya, T. Yasuda, and H. Watanabe. "Yersinia pseudotuberculosis blocks the phagosomal acidification of B10.A mouse macrophages through the inhibition of vacuolar H(+)-ATPase activity." _Microb Pathog_ 27 (1999): 253-63. {{< br >}}{{< br >}} Hayashi, F., T. K. Means, and A. D. Luster. "Toll-like receptors stimulate human neutrophil function." _Blood_ 102 (2003): 2660-9. {{< br >}}{{< br >}} _Recommended Reading_ {{< br >}}{{< br >}} Niedergang, F., and P. Chavrier. "Signaling and membrane dynamics during phagocytosis: many roads lead to the phagos(R)ome." _Curr Opin Cell Biol_ 16 (2004): 422-8. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 8 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} How do multi-cellular organisms destroy pathogens? {{< br >}}{{< br >}} Complement and Reactive Oxygen {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Multi-cellular organisms retained the use of defense mechanisms developed in unicellular organisms and also developed new ones. Deuterostomia introduced a cascade system of peptides, which destroy not only bacterial but other membranes as well. This peptide cascade has evolved differently in disparate groups. In invertebrates, such systems consist of a few peptides, whereas in mammals, there are about 30 proteins involved in the complement cascade, and it is highly regulated. In multi-cellular organisms, specialized immune cells have also introduced the use (in addition to anti-microbial peptides) of reactive oxygen species, such as O2-, H2O2, and NO to kill pathogens. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Kotwal, G. J., S. N. Isaacs, R. McKenzie, M. M. Frank, and B. Moss. "Inhibition of the complement cascade by the major secretory protein of vaccinia virus." _Science_ 250 (1990): 827-30. {{< br >}}{{< br >}} Singhrao, S. K., J. W. Neal, N. K. Rushmere, B. P. Morgan, and P. Gasque. "Spontaneous classical pathway activation and deficiency of membrane regulators render human neurons susceptible to complement lysis." _Am J Pathol_ 157 (2000): 905-18. {{< br >}}{{< br >}} _Recommended Reading_ {{< br >}}{{< br >}} Nappi, A. J., E. Vass, F. Frey, and Y. Carton. "Nitric oxide involvement in Drosophila immunity." _Nitric Oxide_ 4 (2000): 423-30. {{< br >}}{{< br >}}  |
| {{< td-colspan 4 >}} {{< br >}}{{< br >}} **Part III: An Evolutionary Milestone: The Origin of an Adaptive Immune System** {{< br >}}{{< br >}} {{< /td-colspan >}} ||||
|  {{< br >}}{{< br >}} 9 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Receptors of the Adaptive Immune System {{< br >}}{{< br >}} Antigen Recognition by Cells of the Adaptive Immune System Involves The Major Histocompatibility Complex (MHC) {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Receptors of the adaptive immune system originated from the innate immune receptors of the immunoglobulin super-family. Multiple genes that have likely evolved by gene duplication encode these receptors. In the genome, they are represented by several fragments, which can randomly rearrange to create a enormous number of diverse receptors. These receptors can be further diversified somatically by mutations and gene conversion. Proteins performing the rearrangement are thought to be bacterial transposons that became integrated into vertebrate genome. Adaptive immunity represents a new level of the evolution of immune system, where recognition of a foreign molecule (antigen) and "self" markers (MHC) are combined in one mechanism. T cells of the adaptive immune system recognize foreign antigens only in association with molecules of the Major Histocompatibility Complex (MHCI and II), which are expressed on the surface of antigen presenting cells. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Hozumi, N., and S. Tonegawa. "Evidence for somatic rearrangement of immunoglobulin genes coding for variable and constant regions." _PNAS_ 73 (1976): 3628-32. {{< br >}}{{< br >}} Kovacsovics-Bankowski, M., and K. Clark, et al. "Efficient major histocompatibility complex class I presentation of exogenous antigen upon phagocytosis by macrophages." _PNAS_ 90 (1993): 4942-6. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 10 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Clonal Selection and Immunological Memory {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} The extensive variability of receptors of the adaptive immune system in combination with clonal selection improves the efficiency of an immune response. The memory of an antigen gets encoded in the clonal composition of the organism's immune cells instead of being conservatively encoded in the genome. Then, adaptive immunity is capable of responding to an unlimited variety of new antigens throughout the life course of the organism. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Jerne, N. K., and P. Avegno. "The development of the phage-inactivating properties of serum during the course of specific immunization of an animal." _J Immunol_ 76 (1956): 200-5. {{< br >}}{{< br >}} Burnet, F. M. "A modification of Jerne's theory of antibody production using the concept of clonal selection." _CA Cancer J Clin_ 26 (1976): 119-21. {{< br >}}{{< br >}} Pauling, L. "A Theory of the Structure and Process of Formation of Antibodies." _J Am Chem Soc_ 62 (1940): 2643-57. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 11 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} How does the adaptive immune system interact with the innate immune system? {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} The adaptive immune system originated from the innate system and is functionally dependent on it. The introduction of adaptive mechanisms made the whole immune system of vertebrates a very diversified and complicated structure with many specialized cellular and humoral components. Effective communication is crucial for functioning of complex systems, thus many ways of communication between the innate and adaptive immune systems, as well as within them, have evolved. The major role in this communication is played by cytokines. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Hsieh, C. S., and S. E. Macatonia, et al. "Development of TH1 CD4+ T cells through IL-12 produced by Listeria-induced macrophages." _Science_ 260 (1993): 547-9. {{< br >}}{{< br >}} Rimoldi, M., and M. Chieppa, et al. "Intestinal immune homeostasis is regulated by the crosstalk between epithelial cells and dendritic cells." _Nat Immunol_ 6 (2005): 507-514. {{< br >}}{{< br >}} _Recommended Reading_ {{< br >}}{{< br >}} Matzinger, P. "The danger model: a renewed sense of self." _Science_ 296 (2002): 301-5. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 12 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Change or Die {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Pathogens evolve constantly developing new ways of escaping immune surveillance, which means that the immune system must change as well. The evolution of the immune system is always co-evolution with pathogens. For example, T cells evolved to fight pathogens; then pathogens such as HIV capable of surviving in T cells evolved. Viruses often survive by manipulating the host defense machinery, for example, inducing the synthesis of anti-apoptotic proteins in the host cells they infect. The race between immune system and pathogens never ends. {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Dean, M., and M. Carrington, et al. "Genetic restriction of HIV-1 infection and progression to AIDS by a deletion allele of the CKR5 structural gene." _Science_ 273 (1996): 1856-62. {{< br >}}{{< br >}} Tram, U., and W. Sullivan. "Role of delayed nuclear envelope breakdown and mitosis in Wolbachia-induced cytoplasmic incompatibility." _Science_ 296, no. 5570 (2002): 1124-6. {{< br >}}{{< br >}}  |
|  {{< br >}}{{< br >}} 13 {{< br >}}{{< br >}}  |  {{< br >}}{{< br >}} Student Oral Presentations {{< br >}}{{< br >}} Course Evaluations {{< br >}}{{< br >}}  | &nbsp; |